src/torus.ts

```diff
@@ -493,6 +489,15 @@ class Torus {
         } else if (typeOfUser === "v2") {
           isUpgraded = metadataNonce.eq(new BN("0"));
         }
+
+        // deriving address from pub key coz pubkey is always available
+        // but finalPrivKey won't be available for  v2 user upgraded to 2/n
+        let finalEvmAddress = "";
+        if (finalPubKey) {
```

Comment:
in wht scenario finalPubKey won't be available ?

Reply:
the bug scenario where metadata doesn't come